docs/src/modules/components/DemoSandboxed.js

```diff
@@ -121,8 +121,21 @@ DemoFrame.propTypes = {
 
 // Use the default MUI theme for the demos
 const getTheme = (outerTheme) => {
+  const brandingDesignTokens = getDesignTokens(outerTheme.palette.mode);
+  const isCustomized =
+    outerTheme.palette.primary?.main &&
+    outerTheme.palette.primary.main !== brandingDesignTokens.palette.primary.main;
   const resultTheme = createTheme(
-    { palette: { mode: outerTheme.palette.mode || 'light' } },
+    {
+      palette: {
+        mode: outerTheme.palette.mode || 'light',
+        ...(isCustomized && {
+          // Apply color from the color playground
+          primary: { main: outerTheme.palette.primary.main },
+          secondary: { main: outerTheme.palette.secondary.main },
```

Comment:
Need to change to this so that `light` and `dark` are recalculated based on the `main` color.